internal/services/v1/permissions.go

```diff
@@ -785,6 +780,14 @@ func (a *loadBulkAdapter) Next(_ context.Context) (*tuple.Relationship, error) {
 	a.current.Subject.ObjectID = a.currentBatch[a.numSent].Subject.Object.ObjectId
 	a.current.Subject.Relation = stringz.DefaultEmpty(a.currentBatch[a.numSent].Subject.OptionalRelation, tuple.Ellipsis)
 
+	if a.currentBatch[a.numSent].OptionalCaveat != nil {
+		a.caveat.CaveatName = a.currentBatch[a.numSent].OptionalCaveat.CaveatName
+		a.caveat.Context = a.currentBatch[a.numSent].OptionalCaveat.Context
+		a.current.OptionalCaveat = &a.caveat
+	} else {
+		a.current.OptionalCaveat = nil
```

Comment:
same here, and the expires branch as well

Reply:
Same